esmvaltool/cmorizers/data/downloaders/datasets/jra_55.py

```diff
@@ -0,0 +1,115 @@
+"""Script to download JRA-55 from RDA."""
+import logging
+import os
+
+from datetime import datetime
+
+from dateutil import relativedelta
+
+from esmvaltool.cmorizers.data.downloaders.wget import WGetDownloader
+
+
+logger = logging.getLogger(__name__)
+
+
+def download_dataset(config, dataset, dataset_info, start_date, end_date,
+                     overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    dataset_info : dict
+         Dataset information from the datasets.yml file
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    downloader = WGetDownloader(
+        config=config,
+        dataset=dataset,
+        dataset_info=dataset_info,
+        overwrite=overwrite,
+    )
+
+    os.makedirs(downloader.local_folder, exist_ok=True)
+
+    user = os.environ.get("rda-user")
```

Comment:
Authentication methods may have changed on RDA, my login uses my orcid login, so this user, password wasn't working for me. But I commented out all the `downloader.login()` (lines 43-56, 64-69) stuff and was able to download fine on Gadi supercomputer.